database/migrations/create_state_history_table.php.stub

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateStateHistoryTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('state_history', function (Blueprint $table) {
+           $table->increments('id');
+           $table->string('transition');
+           $table->string('to');
+           $table->integer('actor_id')->nullable();
+           $table->integer('statable_id');
```

Comment:
You could replace the `statable_*` columns with `$table->morphs('statable')`.

Reply:
Hey @sebdesign - It seems that the requested changes are made - What do you think, is it possible to merge the PR?